File: api/GroceryList/GroceryList.Domain/Item.cs
namespace GroceryList.Domain;

public class Item
{
    public int Id { get; set; }
    public int CartId { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public decimal? PricePerUnit { get; set; }
    public decimal? TotalPrice
    {
        get
        {
            return PricePerUnit * Quantity;
        }
        private set
        {

        }
    }
    public int Quantity { get; set; }
    public string? imgUrl { get; set; }
    public string? img { get; set; }
    public int? AlternativeItemId { get; set; }
    public bool IsPrimary { get; set; }
    public int Status { get; set; }
}

File: app/app/src/main/java/com/kotlin101/group2/grocerylist/data/api/models/Item.kt
package com.kotlin101.group2.grocerylist.data.api.models

 data class Item(
    var alternativeItemId: Int,
    var cartId: Int,
    var description: String,
    var id: Int,
    var img: Any?,
    var imgUrl: Any?,
    var isPrimary: Boolean,
    var name: String,
    var pricePerUnit: Double,
    var quantity: Int,
    var status: Int,
    var totalPrice: Double
)